hook-consumer/src/config.rs

```diff
@@ -72,4 +72,7 @@ pub struct RetryPolicyConfig {
 
     #[envconfig(default = "100000")]
     pub maximum_interval: EnvMsDuration,
+
+    #[envconfig(default = "default")]
+    pub retry_queue_name: String,
```

Comment:
Unsure if we should configure retries to a separate queue as a default.

Reply:
> two sets of consumers

I wonder if we should make it easier on ourselves and make consumers handle multiple queues (I'm assuming it could be part of the dequeue `ORDER BY`)? Maybe that would make observability more annoying, though. Just thinking out loud.

Reply:
I wonder that too... Maybe we should invest some time on writing a benchmarking binary that we can run and test ideas on.

Reply:
At least to get some performance numbers. "Benchmarking observability" sounds more abstract...

Reply:
I'll go ahead and merge for now, we can revisit these ideas in the future as they could have potential.